docs/03.API-EXAMPLE.md

```diff
@@ -551,6 +551,32 @@ Value of x is 12
 Value of x is 17
 ```
 
+## Changing the seed of pseudorandom generated numbers
+
+If you want to change the seed of `Math.random()` generated numbers, you have to initialize the seed value with `srand`.
+A recommended method is using `jerry_port_get_current_time()` or something based on a constantly changing value, therefore every run produces truly random numbers.
+
+[doctest]: # ()
+
+```c
+#include <string.h>
+#include <stdlib.h>
+#include "jerryscript.h"
+#include "jerryscript-port.h"
+
+int
+main (void)
+{
+  const jerry_char_t script[] = "var a = Math.random ();";
```

Comment:
The other thing that bothers me is that the example code is a bit "alone", i.e., it does not build on the info introduced by the previous steps. At least a print should be added to the example script so that when someone really copies this code out into a source file and links it to jerry, they get some output (a value changing randomly on each run) on display. It's not as simple as a `jerry_run_simple` ofc, but not that complex either. Something along the lines:

```c
  /* Initialize engine */
  jerry_init (JERRY_INIT_EMPTY);

  /* Register 'print' function from the extensions */
  jerryx_handler_register_global ((const jerry_char_t *) "print",
                                  jerryx_handler_print);
  /* Evaluate script */
  jerry_value_t eval_ret = jerry_eval (script, script_size, false);

  /* Free JavaScript value, returned by eval */
  jerry_release_value (eval_ret);

  /* Cleanup engine */
  jerry_cleanup ();

  return 0;
```

(+ `print` in the script and the `srand` call ofc (preferably with a comment highlighting what's happening), and it may also need the inclusion of `"jerryscript-ext/handler.h"`)